pkg/apis/componentconfig/types.go

```diff
@@ -56,4 +56,8 @@ type ControllerManagerConfiguration struct {
 	// ResyncInterval is the interval on which the controller should re-sync
 	// all informers.
 	ResyncInterval time.Duration
+
+	// Whether or not to send the proposed optional
+	// OpenServiceBroker API Context Profile field
+	OSBAPIContextProfile bool
```

Comment:
Thanks for adding the flag.

In k8s, there was a recent decision (which has not made it into doc yet, as far as I know -- I'll create an issue) to treat alpha fields like so:

1.  Prefix the name of the field with `Alpha`
2.  When the field progresses to beta, drop the `Alpha` prefix; no data migration

I think this counts as alpha because it has not been adopted.  Thoughts @derekwaynecarr ?